### PR TITLE
[WIP] Example buildings to be JSONized

### PR DESCRIPTION
**Pull request by SunshineDistillery**
SUMMARY: Mods "Even more buildings."

These are some example buildings for #21824. It isn't meant to be merged as is; it's in mod form so it's easy to load up and examine the sites. 

Rather than just remake cube buildings, I altered them a bit to make them more interesting/realistic.  The Missile Silo currently lacks poison gas, a launch computer, and a missile. 

For the real thing, I can condense them to single files or leave them as they are.

### CODE REVIEW

File: data/mods/mapspecials_demo/missile_silo/silo_blc_01.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+",": "t_grass",
+"%": "t_fence_barbed"
+},
+"monster": {
+"Z": { "monster": "mon_zombie_soldier" },
+"z": { "monster": "mon_turret_rifle" }
+}
+}
+}
+]
```
Comment (Coolthulhu) [inline data/mods/mapspecials_demo/missile_silo/silo_blc_01.json]: Could use styling

**Comment by Coolthulhu** [comment]
Missile computer should be jsonable now.

**Comment by BorkBorkGoesTheCode** [comment]
Are you here @SunshineDistillery ?

**Comment by illi-kun** [comment]
@ZhilkinSerg it seems that you have some experience with implementation of similar changes and it also seems that this PR is obsolete. It would be really great if you can re-implement this in bunch of smaller PRs (with possible changes, of course, it's totally up to you).

**Comment by ZhilkinSerg** [comment]
> @ZhilkinSerg it seems that you have some experience with implementation of similar changes and it also seems that this PR is obsolete. It would be really great if you can re-implement this in bunch of smaller PRs (with possible changes, of course, it's totally up to you).

I will pick this up soon.

**Comment by ZhilkinSerg** [comment]
I've added computers and poison gas and linted whole mod in #26030.

**Pull request merged**